test/unit/test_helpers_tests.rb

```diff
@@ -50,6 +50,24 @@ class UnitTests < Assert::Context
       assert_equal exp, subject.params[Dk::ABDeploy::RELEASE_B_DIR_PARAM_NAME]
     end
 
+    should "setup the params the update task does" do
+      subject.set_dk_abdeploy_validate_params
```

Comment:
@jcredding we discussed how I can't automatically do this as the user needs to control when the validate helper is called b/c it is destructive to any validate params that may already be set.  We discussed switching the helper to `||=` everything to not be destructive.  We also discussed switching to task specific test helpers and letting much-plugin make sure stuff isn't called multiple times.  I'll go with this option in another effort.